### PR TITLE
Show successful tests

### PR DESCRIPTION
**Pull request by tyll**
This addresses #53

### CODE REVIEW

File: pkgdb2client/admin.py [hunk 1]
```diff
@@ -249,8 +249,12 @@ def _ask_what_to_do(messages):
...
     what to do, ie: approve, deny, pass
     '''
-    for message in messages:
-        print message
+    for message in messages["good"]:
+        print("[+] " + message)
+    for message in messages["bad"]:
+        print("[-] " + message)
+    if not messages["bad"]:
+        print("[+] " + "all checks were good")
```
Comment (pypingou) [inline pkgdb2client/admin.py]: Maybe we should break the layout there, just to make it easier for admins processing the requests to see it

Comment (tyll) [inline pkgdb2client/admin.py]: For me this output looks readable:
```
#1865 (Awaiting Review) - ignatenkobrain requested the new package "python-PyLink" on "master"
       review: https://bugzilla.redhat.com/1280482
FAS password for user till: 
[+] Summary of bug 1280482 is: Review Request: python-PyLink - Universal communication interface using File-Like API
[+] Review approved by packager zbyszek@in.waw.pl
[+] Requester ignatenkobrain is a packager
[+] all checks were good

```

Comment (pypingou) [inline pkgdb2client/admin.py]: what about:
```
#1865 (Awaiting Review) - ignatenkobrain requested the new package "python-PyLink" on "master"
       review: https://bugzilla.redhat.com/1280482
FAS password for user till: 
[+] Summary of bug 1280482 is: Review Request: python-PyLink - Universal communication interface using File-Like API
[+] Review approved by packager zbyszek@in.waw.pl
[+] Requester ignatenkobrain is a packager
==> All checks were good
```

**Comment by pypingou** [comment]
Btw, looks like you'll need to rebase :)

**Comment by pypingou** [comment]
:+1: for me

**Pull request merged**